### PR TITLE
Confirmation page with visual design

### PR DESCRIPTION
**Pull request by adborden**
- [ ] remove the confirmation test page (or stick it to a development flag)

### CODE REVIEW

**Comment by hbillings** [comment]
Current state: 
screencapture-localhost-4000-request-agency-component-db0d3e3f-7280-4a42-864f-94aa90f5c4e8-1509591760773

Left to do: 
- [ ] Add print button
- [ ] Add today's date near request number

**Comment by hbillings** [comment]
@adborden It looks like some of the `RequestSummarySection` chai tests are failing. I may need a hand figuring out why.

File: js/pages/confirmation.jsx
Comment (hbillings) [inline js/pages/confirmation.jsx]: @adborden This was to keep eslint from complaining that we were mixing absolute and relative imports. I figured you probably cared more about the grouping of imports than absolute/relative, but I'm happy to change if I made the wrong call!

File: js/components/confirmation.jsx
Comment (adborden) [inline js/components/confirmation.jsx:11]: I haven't used this yet, but it looks like browser support is pretty good these days
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString

File: js/components/confirmation.jsx
Comment (adborden) [inline js/components/confirmation.jsx]: I would prefer not to mix/match abbreviations and just use the full month name if possible.

File: js/components/confirmation.jsx
Comment (adborden) [inline js/components/confirmation.jsx]: I've seen some agencies with website === null, so we have to put in checks for any object or array fields:
`{ agencyComponet.website && ... }`

File: js/components/request_summary_contact_section.jsx
Comment (adborden) [inline js/components/request_summary_contact_section.jsx]: these fields should be: `address_line1`, `address_line2`, `address_city`, `address_state_province`, `address_zip_postal_code`.

Let's also include country below. Refer to https://github.com/18F/beta.foia.gov/blob/develop/js/util/request_form/form_sections.js#L11 for the new field names.

File: js/components/request_summary_contact_section.jsx
Comment (adborden) [inline js/components/request_summary_contact_section.jsx]: These fields should be `name_first`, `name_last`.

**Comment by adborden** [comment]
When the sections are empty, we shouldn't be showing them in the summary, right?

**Comment by adborden** [comment]
har har, since I created this PR, I cannot approve it, but it has my approval!
☑️

File: js/components/confirmation.jsx
Comment (hbillings) [reply js/components/confirmation.jsx]: Ha, AP style habits die hard. Will update!

File: js/components/request_summary_contact_section.jsx
Comment (hbillings) [reply js/components/request_summary_contact_section.jsx]: `name_first` and `name_last` were returning blank for me, while these returned data. I'm not clear on what needs to change to make this work.

Comment (adborden) [reply js/components/request_summary_contact_section.jsx]: yeah, this has been a long awaited change. First, we were waiting on https://github.com/usdoj/foia-api/pull/97 to be merged, which required some back and forth and review with OIP and design. Even now that it's merged, we still need to create the new forms on the back stage, something we could only have done _after_ 97 was merged.

OIP said they would be adding a few forms from the survey results on Monday/Tuesday and this will have the proper fields. Of course, an agency could come in and wreck their form if they chose to do so.

File: js/components/confirmation.jsx
Comment (hbillings) [reply js/components/confirmation.jsx:11]: It looks like in order to specify a format other than the default, you have to also pass in a language option (without that, at least, the `options` do nothing in my console). I don't want to format the date as `month/day/year` (the default) because that will be confusing outside of the US. :-/

**Review by hbillings — APPROVED** [review]

**Pull request merged**
